_docs/advanced-setup/html5.md

```diff
@@ -34,6 +34,38 @@ To use an external JavaScript library with Lime, you need [Haxe externs](https:/
 
 Members of the Haxe community have already created open source externs for a number of popular JavaScript libraries, so be sure to search [Haxelib](https://lib.haxe.org/search) or [Github](https://github.com/search) to see if JS externs exist for your library before you try to create your own.
 
+### Adding dependencies
+
+The way you add external libraries can vary. Some Haxe externs are configured to use with Lime out of the box (like [haxe-simple-peer](https://github.com/ixiagames/haxe-simple-peer)), so you just use it with `<haxelib name="simple-peer" />`. Other externs may require to include appropriate js dependencies by you. There are three ways for doing so:
```

Comment:
That's because there is an include.xml with the dependency tag inside the haxe lib. Maybe it's best to be more explicit because it's a bit vague here.

Reply:
Yeah maybe something like: 

The way you add external libraries can vary. Some Haxe externs are configured to use with Lime out of the box thanks to their [include.xml file](https://lime.openfl.org/docs/project-files/xml-format/#includexml), so you just use it with `<haxelib name="simple-peer" />`. Other externs may require to include appropriate js dependencies by you. There are three ways for doing so: